apps/web/src/ee/clerk/pages/ManageAccountPage.css

```diff
@@ -0,0 +1,83 @@
+.mantine-Modal-modal {
+  border-radius: 12px;
+  background-color: var(--nv-colors-surface-panel);
+  padding: 0px;
+  min-height: 100%;
+  max-height: 100%;
+}
+
+.mantine-Tabs-panel {
+  padding: var(--semantic-paddings-popover-vertical, 24px) var(--semantic-paddings-popover-horisontal, 24px);
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--semantic-margins-layout-PageTitleBottom, 24px);
+  flex: 1 0 0;
+  align-self: stretch;
+  border-radius: 12px;
+  background: var(--nv-colors-surface-panel-subsection);
+  overflow-y: auto;
+  /* TODO: these 2 are wrong, need to calculate based on available parent size */
```

Comment:
Help needed, I don't know how to calculate this values below to be based on available space in parent

Reply:
If the parent is display: flex, does a height: 100% solve the issue? We can pair on this later today.

Reply:
Didn't work. I will ask Joel to help me with this